main.py

```diff
@@ -0,0 +1,51 @@
+originArray = [[1,2,3],[4,5,6],[7,8,9]]
+finalArray = []
+
+# append to final array leftmost column of origin array
+def goUp(a):
+    for i in range(0, len(a)):
```

Comment:
Look at the documentation for the "range" function. It can get from 1 up to 3 parameters.
What's the difference between all of the following codes?
```
> range(3)
> range(6)
> range(3, 6)
> range(3, 9, 2)
```
How can you write this line better?